src/main/java/competition/subsystems/pose/PoseSubsystem.java

```diff
@@ -30,4 +37,52 @@ protected double getRightDriveDistance() {
         return drive.getRightTotalDistance();
     }
 
+
+    private SwerveDrivePoseEstimator initializeSwerveOdometry() {
+        return new SwerveDrivePoseEstimator(
+                drive.getSwerveDriveKinematics(),
+                getCurrentHeading(),
+                new SwerveModulePosition[] {
```

Comment:
I think there's a method on the drive subsystem to get these all at once?

Reply:
Unable to find, simplified using getSwerveModulePositions method instead.